docs/specs/adminbundle/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>TripGo Mobility Admin Bundle API (Beta)</title>
```

Comment:
This says beta but points at api.tripgo.com. Should we remove the beta here then?

```suggestion
    <title>TripGo Mobility Admin Bundle API</title>
```